plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergSplitSource.java

```diff
@@ -88,7 +90,13 @@ protected QueryRunner createQueryRunner()
         this.metastore = createTestingFileHiveMetastore(metastoreDir);
         this.operationsProvider = new FileMetastoreTableOperationsProvider(new HdfsFileIoProvider(hdfsEnvironment));
 
-        return createIcebergQueryRunner(ImmutableMap.of(), ImmutableMap.of(), ImmutableList.of(NATION), Optional.of(metastoreDir));
+        return createIcebergQueryRunner(ImmutableMap.of(), ImmutableMap.of(), Optional.of(metastoreDir), this::initializeSchema);
+    }
+
+    private void initializeSchema(QueryRunner queryRunner)
```

Comment:
`createIcebergQueryRunner` should be able to create the schema for the typical case

Reply:
It can there is a call with most parameters defaulted `createIcebergQueryRunner()`